pkg/lisp/go.mod

```diff
@@ -4,12 +4,11 @@ go 1.12
 
```

Comment:
This file must have gotten changed by accident, and is not part of this PR. This should be reverted.